profiles/common/modules/features/solr_config/solr_config.module

```diff
@@ -6,26 +6,34 @@
 
 include_once 'solr_config.features.inc';
 
+/**
+ * Implements hook_apachesolr_exclude().
+ */
 function solr_config_apachesolr_exclude($entity_id, $entity_type, $row, $env_id) {
   if ($entity_type == 'file') {
     $node = node_load($row->parent_entity_id);
 
-    if (isset($node->workbench_moderation)) { // node moderate with workbench moderation
+    if (isset($node->workbench_moderation)) {
+      // Node moderate with workbench moderation.
       $workflow = $node->workbench_moderation;
-      if (isset($workflow['published'])) { // node parent is published
+      if (isset($workflow['published'])) {
+        // Node parent is published.
         return FALSE;
       }
       else {
-        apachesolr_index_delete_entity_from_index($env_id, $entity_type, $entity_id . "-node-" . $node->nid); // remove file from solr index
+        apachesolr_index_delete_entity_from_index($env_id, $entity_type, $entity_id . "-node-" . $node->nid);
+        // Remove file from solr index.
         drupal_set_message(t('Entity @type @id removed from solr index', array('@type' => $entity_type, '@id' => $entity_id)));
         return TRUE;
       }
     }
-    elseif ($node->status == 1) { // standard drupal workflow : node parent is published
+    elseif ($node->status == 1) {
+      // Standard drupal workflow : node parent is published.
```

Comment:
Move documentation one line higher, this occurs several times more in this file. Do you think it would be good to fix this in the Coder module?